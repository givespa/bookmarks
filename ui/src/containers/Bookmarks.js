import { BookmarkForm } from '../components/form';
import { BookmarkList } from '../components/list';

// Note: API calls should be done here, 'axios' is already added as dependency

export const Bookmarks = () => (
    <>
        <BookmarkForm />
        <BookmarkList />
    </>
);