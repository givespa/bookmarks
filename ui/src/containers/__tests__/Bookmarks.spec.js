import { shallow } from 'enzyme';

import { Bookmarks } from '../Bookmarks';

describe('Bookmarks', () => {
    it('should match snapshot', () => {
        const rendered = shallow(<Bookmarks />);
        expect(rendered).toMatchSnapshot();
    });
});