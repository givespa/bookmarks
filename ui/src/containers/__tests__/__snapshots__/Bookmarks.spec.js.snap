// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Bookmarks should match snapshot 1`] = `
<Fragment>
  <BookmarkForm />
  <BookmarkList />
</Fragment>
`;
